modules/ROOT/pages/_partials/sync/ref-data-sync-terminology.adoc

```diff
@@ -0,0 +1,16 @@
+[id='ref-data-sync-terminology-{context}']
+= Data Sync terminology
+
+This section describes terminology that is associated with Data Sync.
+
+.Data Sync terminology
+Data Sync framework:: Data Sync is a JavaScript framework that enables a developer to add real time data synchronization to both mobile and web clients.
+The Data Sync framework comprises of two components, the _Sync Server_ and the _Sync Client_.
```

Comment:
@jstaffor  I think this material is good, but not the server/client bits

Reply:
Change made in https://github.com/aerogear/mobile-docs/pull/758/commits/14ca4978d6707971ffa965a468241386dafbf56a